src/components/AttachmentCarousel/index.js

```diff
@@ -176,6 +176,7 @@ class AttachmentCarousel extends React.Component {
         _.forEach(actions, (action) => htmlParser.write(_.get(action, ['message', 0, 'html'])));
         htmlParser.end();
 
+        attachments = this.canUseTouchScreen ? attachments.reverse() : attachments;
```

Comment:
`reverse()` mutates the array, so you just need to wrap it in a condition.

```javascript
if (this.canUseTouchScreen) {
  attachments.reverse();
}
```

Reply:
Actually both options are correct - based on the docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse it's indeed reversing the array in place but also returns the reversed array...

Reply:
But I think I'll just re-write it to use the `toReversed` method (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toReversed) to avoid this mutation.

Reply:
Yes, your way it's valid, but you had to change `attachments` to `let`, put this ternary to reverse the list and assign it again to `attachment`. There is no need for all this if you just put the condition and reverse it.

Example of a file which did the same thing: https://github.com/Expensify/App/blob/main/src/libs/SidebarUtils.js#L178